cli/toolchain_cmd.go

```diff
@@ -513,7 +469,7 @@ func installRubyToolchain() error {
 	srclibpathDir := filepath.Join(strings.Split(srclib.Path, ":")[0], toolchain) // toolchain dir under SRCLIBPATH
 
 	if _, err := exec.LookPath("ruby"); isExecErrNotFound(err) {
-		return skippedToolchain{toolchain, "no `ruby` in PATH (assuming you don't have Ruby installed and you don't want the Ruby toolchain)"}
+		return errors.New("no `ruby` in PATH (assuming you don't have Ruby installed and you don't want the Ruby toolchain)")
```

Comment:
Isn't this error message going to cause the confusion we're trying to avoid? User asks us to install Ruby explicitly...and we say we assume they don't want Ruby installed.